include/iceoryx_dds/dds/fast_data_reader.hpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2020 - 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
I think in nearly all files the copyright header is wrong.

You can remove the lines:

```
// Copyright (c) 2020 - 2021 by Robert Bosch GmbH. All rights reserved.
// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

And replace it with a statement from you
```
// Copyright (c) 2021 by MY_COMPANY (or) MY_NAME (or) MY_GITHUB_NAME All rights reserved.
```